### ISSUE TITLE
Compile error in AttentionBank

### ISSUE THREAD

**Issue by linas** [opencog/atomspace#1251]
I now get this:
```
opencog/attentionbank/AttentionBank.h:345:5: warning: no return statement in function returning non-void [-Wreturn-type]
     }
     ^
```

**Comment by linas** [comment]
Also, opencog doesn't build any more.

**Comment by linas** [comment]
Fixed in #1252

**Comment by linas** [comment]
oh, and the other side by https://github.com/opencog/opencog/pull/2748
